packages/components/src/core/Button/index.stories.tsx

```diff
@@ -60,213 +60,352 @@ export const Default = {
-  fontWeight: "normal",
-  gridColumn: "2 / 6",
-  justifySelf: "stretch",
-  margin: "20px 0",
-  paddingTop: 10,
+
+export const RoundedLivePreview = {
+  args: {
+    label: "Label",
+  },
+  parameters: {
+    snapshot: {
+      skip: true,
+    },
+  },
+  render: (args: Args) => <RoundedLivePreviewDemo {...args} />,
 };
-const iconLabel = {
-  alignSelf: "end",
-  borderStyle: "solid none none none",
-  borderWidth: "1px",
-  fontWeight: "normal",
-  gridColumn: "2 / 6",
-  justifySelf: "stretch",
-  margin: "0 0 5px 0",
-  paddingTop: 10,
+
+// Square Live Preview
+
+const SquareLivePreviewDemo = (props: Args): JSX.Element => {
+  return (
+    <div style={placementStyles as React.CSSProperties}>
+      <RawButton {...props} sdsStyle="square" sdsType="primary">
+        {TEXT}
+      </RawButton>
+      <RawButton
+        {...props}
+        startIcon={<Icon sdsIcon="download" sdsSize="s" sdsType="button" />}
+        sdsStyle="square"
+        sdsType="primary"
+      >
+        {TEXT}
+      </RawButton>
+      <RawButton {...props} sdsStyle="square" sdsType="secondary">
+        {TEXT}
+      </RawButton>
+      <RawButton
+        {...props}
+        startIcon={<Icon sdsIcon="download" sdsSize="s" sdsType="button" />}
+        sdsStyle="square"
+        sdsType="secondary"
+      >
+        {TEXT}
+      </RawButton>
+    </div>
+  );
 };
-const disabledLabel = {
-  alignSelf: "end",
-  fontWeight: "normal",
-  gridColumn: "1 / 2",
-  marginTop: 0,
+
+export const SquareLivePreview = {
+  args: {
+    label: "Label",
+  },
+  parameters: {
+    snapshot: {
+      skip: true,
+    },
+  },
+  render: (args: Args) => <SquareLivePreviewDemo {...args} />,
 };
-const stateLabel = {
-  fontWeight: "normal",
-  margin: "10px 0",
+
+// Minimal Live Preview
+
+const MinimalLivePreviewDemo = (props: Args): JSX.Element => {
+  const minimalPlacementStyles: React.CSSProperties = {
+    display: "grid",
+    gridColumnGap: "24px",
+    gridRowGap: "0px",
+    gridTemplateColumns: "repeat(3, min-content)",
+    gridTemplateRows: "1fr",
+  };
+
+  return (
+    <div style={minimalPlacementStyles}>
+      <RawButton {...props} sdsStyle="minimal" sdsType="primary">
+        {TEXT}
+      </RawButton>
+      <RawButton
+        {...props}
+        startIcon={<Icon sdsIcon="download" sdsSize="s" sdsType="button" />}
+        sdsStyle="minimal"
+        sdsType="primary"
+      >
+        {TEXT}
+      </RawButton>
+      <RawButton {...props} sdsStyle="minimal" sdsType="secondary">
+        {TEXT}
+      </RawButton>
+    </div>
+  );
 };
 
-export const LivePreview = {
+export const MinimalLivePreview = {
+  args: {
+    label: "Label",
+  },
   parameters: {
     snapshot: {
       skip: true,
     },
   },
-  render: (props: Args): JSX.Element => {
-    // loop through all SDS_STYLES
+  render: (args: Args) => <MinimalLivePreviewDemo {...args} />,
+};
+
+// Screenshot Test
+
+const ScreenshotTestDemo = (props: Args): JSX.Element => {
```

Comment:
Create a "...Demo" component which is then called by the main test component (line 396)